common/src/main/java/com/alibaba/nacos/common/utils/ConvertUtils.java

```diff
@@ -16,15 +16,17 @@
 
 package com.alibaba.nacos.common.utils;
 
-import org.apache.commons.lang3.StringUtils;
-
 /**
  * @author <a href="mailto:liaochuntao@live.com">liaochuntao</a>
  */
 public final class ConvertUtils {
 
     private static final String NULL_STR = "null";
 
+    public static int toInt(String val) {
```

Comment:
You may need to add a comment, which automatically defaults to 0

Reply:
Okay, I will add a comment in next commit.